app/assets/stylesheets/application.css.scss

```diff
@@ -30,11 +31,11 @@ body[data-action="devise/sessions-new"], body[data-action="devise/sessions-creat
 
 
 .post {
-  background-color: $post-bg-color;
-  border-radius: 10px;
-  list-style: none;
-  margin: 20px 10px;
+  background-color: $white;
+  border-radius: 5px;
```

Comment:
Properties should be ordered background-color, border, border-radius, margin, padding